src/omen/configs/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "internal": {
-        "version": "v1.1.0"
+        "version": "v1.1.1"
     },
     "authentication": {
         "gw2_token": "84CF4DFD-D23B-9D4D-A94D-918A33DD0F8FA692141E-717C-440B-94EA-485BE23F3D1D"
```

Comment:
public api token?

Reply:
read only api token and contains no sensitive info